src/sms/ports/inbound/docs_handler.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 - 2024 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains the abstract definition of a DocsHandler."""
+
+from abc import ABC, abstractmethod
+
+from sms.models import Criteria, DocumentType, UpsertionDetails
+
+
+class DocsHandlerPort(ABC):
+    """Port definition for a Doc Handler"""
```

Comment:
```suggestion
    """Port definition for a MongoDB Document Handler"""
```

Reply:
Fixed in [f71217c](https://github.com/ghga-de/state-management-service/pull/1/commits/f71217c95008826977207d0f7c25f20665896c95)